### PR TITLE
Added Landing Page, Footer, and About Page

### PR DESCRIPTION
**Pull request by Wsoukkachang**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -371,6 +389,25 @@ form .navbarContainer {
...
+/* Landing Page Image  */
+
+.ui.image {
+  padding-top: 50px;
+}
+
+/* Landing Page Image Hover Effect  */
+
+.ui.image img:hover {
+  -webkit-filter: invert(1);
```
Comment (Tarmstrong95) [inline src/App.css]: I would ditch the invert

Comment (Tarmstrong95) [reply src/App.css]: also I would crop the images to have a 1:1 aspect ration.

Comment (Tarmstrong95) [reply src/App.css]: also i would change either the name or roll to match the color scheme

Comment (Wsoukkachang) [reply src/App.css]: Took out the invert and cropped the images with this newer commit.

**Pull request merged**